docs/csharp/language-reference/compiler-messages/cs1616.md

```diff
@@ -37,13 +37,13 @@ Option 'option' overrides attribute 'attribute' given in a source file or added
   
  For the example below, assume you have a key file  named `cs1616.snk`. This file could be generated with the command line:  
```

Comment:
a major nit: there's an extra space between "file" and "named"
Also a second nit: "This file could be generated..." --> "You can generate this file...."